lib/rules/label-content-name-mismatch-matches.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Applicability:
+ * This rule applies to any element that has:
+ * - a semantic role that is a widget that supports name from content, and
+ * - visible text content, and
+ * - an aria-label or aria-labelledby attribute.
+ */
+const { aria } = axe.commons;
+const role = aria.getRole(node, { noImplicit: false, fallback: true });
```

Comment:
Because of our accessibility support baseline, axe-core shouldn't support fallback roles for the time being. Also `noImplicit: false` is the default, so you don't need to set this options object at all.